docs/dev/plugins/target-artifact-conventions/target-artifact-conventions.md

```diff
@@ -17,8 +17,43 @@ The purpose of this document is to specify conventions for the target artifact t
 ## Must have
 - `Readme.html` - a Readme file in the root folder of the target artifact is mandatory. It holds the basic information of the target artifact, also if the plugin needs a short usage guide include it in this file.
 - `output` - the `output` - folder holds everything generated during the transformation.
-  - `output/scripts` - `scripts` is a subfolder of `output` it holds executable scripts generated during the transformation, that are needed to deploy the artifact. Scripts are all lowercase and to seperate logical words use `-`.
+  - `output/scripts` - `scripts` is a subfolder of `output` it holds executable scripts generated during the transformation, that are needed to deploy the artifact. In the [scripts](#scripts) section you can find more about script conventions.
 
 ## Recommended
 
 - `docs` - if the transformation artifact needs a more advanced documentation put it into the `docs` folder. And link to it in the `Readme.html`. To allow the user to browse the docs they are written in HTML.
+
+
+## Scripts
+
+As mentioned before our target artifact folder has a `scripts` folder.
+To prevent random names scripts also have to follow conventions.
+
+### Script conventions
```

Comment:
maybe it make sense to distinguish between shell-scripts and other application-scripts like my_app.py.
it could be that there are a lot of different scripts. For this we could add more subfolders

Reply:
As far as I know all the application specific scripts (like implementation artifacts) will be contained in the output directory.
The script directory shall contain scripts regarding building / deploying the transformation result.

Reply:
you're right